### PR TITLE
Adding few commands for vagrant setup on Clear Linux

### PR DESCRIPTION
**Pull request by knangia**
Added command for installing unzip bundle since vagrant.sh requires unzip, else it will error out. Similarly, added command for installing rsync bundle, since Vagrantfile requires it. These bundles need to be added explicitly on some clear images. Also, removed the sudo, since running as normal user works fine.

### CODE REVIEW

**Comment by NitinAtIntel** [comment]
@ahsan518 is it easy for you to verify?

**Comment by ahsan518** [comment]
Testing the patch. Will post the findings soon

**Comment by ahsan518** [comment]
Looks good to me.

File: clr-k8s-examples/vagrant.md [hunk 1]
```diff
@@ -36,18 +36,34 @@ Note, vagrant installation steps were derived from:
 
 On Clear Linux, run these commands
 ```bash
-sudo wget https://raw.githubusercontent.com/AntonioMeireles/ClearLinux-packer/master/extras/clearlinux/setup/libvirtd.sh
+wget https://raw.githubusercontent.com/AntonioMeireles/ClearLinux-packer/master/extras/clearlinux/setup/libvirtd.sh
 chmod +x libvirtd.sh
 ./libvirtd.sh
-sudo wget https://raw.githubusercontent.com/AntonioMeireles/ClearLinux-packer/master/extras/clearlinux/setup/vagrant.sh
+```
```
Comment (jascott1) [inline clr-k8s-examples/vagrant.md]: Can we make dependencies the first task so it is consistent with instructions for other OS?

Comment (knangia) [reply clr-k8s-examples/vagrant.md]: Yes, will do @jascott1

**Pull request closed without merging**